lib/bindings/langs/swift/Sources/BreezSDKLiquid/Task/SwapUpdated.swift

```diff
@@ -81,4 +88,18 @@ class SwapUpdatedTask : TaskProtocol {
             self.displayPushNotification(title: String(format: notificationTitle, payment.amountSat), logger: self.logger, threadIdentifier: Constants.NOTIFICATION_THREAD_SWAP_UPDATED)
         }
     }
+    
+    func notifyPaymentWaitingFeeAcceptance(payment: Payment) {
+        if !self.notified {
+            self.logger.log(tag: TAG, line: "Payment \(payment.txId ?? "") requires fee acceptance", level: "INFO")
```

Comment:
nit: same here with the txId

Reply:
Fix: [0c21c7e](https://github.com/breez/breez-sdk-liquid/pull/634/commits/0c21c7e7cda778277668a9ad3dbf2a317dae65b3)